### PR TITLE
a couple of necessary seccomp enhancements

### PR DESCRIPTION
**Pull request by Polynomial-C**

### CODE REVIEW

File: src/daemon/priv-seccomp.c [hunk 1]
```diff
@@ -172,6 +174,10 @@ priv_seccomp_init(int remote, int child)
 	    (rc = seccomp_rule_add(ctx, SCMP_ACT_ALLOW, SCMP_SYS(recvfrom), 0)) < 0 ||
 	    (rc = seccomp_rule_add(ctx, SCMP_ACT_ALLOW, SCMP_SYS(readv), 0)) < 0 ||
 	    (rc = seccomp_rule_add(ctx, SCMP_ACT_ALLOW, SCMP_SYS(mprotect), 0)) < 0 ||
+	    (rc = seccomp_rule_add(ctx, SCMP_ACT_ALLOW, SCMP_SYS(sendmmsg), 0)) < 0 ||
+	    (rc = seccomp_rule_add(ctx, SCMP_ACT_ALLOW, SCMP_SYS(recvfrom), 0)) < 0 ||
```
Comment (vincentbernat) [inline src/daemon/priv-seccomp.c:178]: Already above?

File: src/daemon/priv-seccomp.c [hunk 2]
```diff
@@ -148,6 +148,7 @@ priv_seccomp_init(int remote, int child)
 	if ((rc = seccomp_rule_add(ctx, SCMP_ACT_ALLOW, SCMP_SYS(read), 0)) < 0 ||
 	    (rc = seccomp_rule_add(ctx, SCMP_ACT_ALLOW, SCMP_SYS(write), 0)) < 0 || /* write needed for */
 	    (rc = seccomp_rule_add(ctx, SCMP_ACT_ALLOW, SCMP_SYS(open), 0)) < 0 ||
+	    (rc = seccomp_rule_add(ctx, SCMP_ACT_ALLOW, SCMP_SYS(lseek), 0)) < 0 ||
```
Comment (vincentbernat) [inline src/daemon/priv-seccomp.c:151]: Already below?

**Comment by Polynomial-C** [comment]
Sorry for the inconvenience. Should all be okay now.

**Comment by vincentbernat** [comment]
Thanks!

**Pull request merged**